cli/flox-rust-sdk/src/flox.rs

```diff
@@ -165,12 +165,46 @@ pub enum FloxhubError {
     InvalidFloxhubBaseUrl(String, #[source] url::ParseError),
 }
 
+use tempfile::TempDir;
+/// Should only be used in the flox crate
+pub fn test_flox_instance() -> (Flox, TempDir) {
+    use std::str::FromStr;
```

Comment:
I think we should drop the implementation in `tests` then (or simply reexport as `pub use super::test_flox_instance as flox_instance` if we dont want to touch all tests that currently import `flox_instance` (can understand too)
would be good to have that represented in the comment rather than cause wonder why there is two test instances

Reply:
Changing to `pub use super::test_flox_instance as flox_instance`